src/frontend/src/pages/FinancePage/CreateReimbursementRequestForm/CreateReimbursementFormView.tsx

```diff
@@ -0,0 +1,230 @@
+import NERFailButton from '../../../components/NERFailButton';
+import NERSuccessButton from '../../../components/NERSuccessButton';
+import { routes } from '../../../utils/routes';
+import { CreateReimbursementRequestFormInput } from './CreateReimbursementRequestForm';
+import PageLayout from '../../../components/PageLayout';
+
+interface CreateReimbursementRequestFormViewProps {
+  allVendors: Vendor[];
+  allExpenseTypes: ExpenseType[];
+  receiptFiles: {
+    file: File;
+  }[];
+  allWbsElements: {
+    wbsNum: WbsNumber;
+    wbsName: string;
+  }[];
+  control: Control<CreateReimbursementRequestFormInput, any>;
+  reimbursementProducts: ReimbursementProductCreateArgs[];
+  receiptAppend: (args: { file: File }) => void;
+  receiptRemove: (index: number) => void;
+  reimbursementProductAppend: (args: ReimbursementProductCreateArgs) => void;
+  reimbursementProductRemove: (index: number) => void;
+  onSubmit: (data: CreateReimbursementRequestFormInput) => void;
+  handleSubmit: UseFormHandleSubmit<CreateReimbursementRequestFormInput>;
+  errors: FieldErrors<CreateReimbursementRequestFormInput>;
+  totalCost: number;
+}
+
+const CreateReimbursementRequestFormView: React.FC<CreateReimbursementRequestFormViewProps> = ({
+  allVendors,
+  allExpenseTypes,
+  allWbsElements,
+  receiptFiles,
+  reimbursementProducts,
+  control,
+  receiptAppend,
+  receiptRemove,
+  reimbursementProductAppend,
+  reimbursementProductRemove,
+  onSubmit,
+  handleSubmit,
+  errors,
+  totalCost
+}) => {
+  const wbsElementAutocompleteOptions = allWbsElements.map((wbsElement) => ({
+    label: wbsPipe(wbsElement.wbsNum) + ' - ' + wbsElement.wbsName,
+    id: wbsPipe(wbsElement.wbsNum)
+  }));
+
+  const ReceiptFileInput = () => (
+    <>
+      <FormLabel>Receipts</FormLabel>
+      <ul>
+        {receiptFiles.map((receiptFile, index) => (
+          <li>
+            <Stack key={receiptFile.file.name} direction="row" justifyContent="space-between">
+              <Typography>{receiptFile.file.name}</Typography>
+              <IconButton onClick={() => receiptRemove(index)}>
+                <Delete />
+              </IconButton>
+            </Stack>
+          </li>
+        ))}
+      </ul>
+    </>
+  );
+
+  return (
+    <form
+      onSubmit={(e) => {
+        e.preventDefault();
+        e.stopPropagation();
+        handleSubmit(onSubmit)(e);
+      }}
+    >
+      <PageLayout title="Create Reimbursement Request" previousPages={[]}>
+        <Grid container spacing={2}>
+          <Grid item container spacing={2} md={6} xs={12}>
+            <Grid item xs={12}>
+              <FormControl fullWidth>
+                <FormLabel>Purchased From</FormLabel>
+                <Controller
+                  name="vendorId"
+                  control={control}
+                  render={({ field: { onChange, value } }) => (
+                    <Select onChange={(newValue) => onChange(newValue.target.value)} value={value} error={!!errors.vendorId}>
```

Comment:
this could be an autocomplete if we think there are gonna be a lot of vendors

Reply:
yeah... but i dont think we are...

Reply:
it's fine for now then, we can always change it later if need be